src/app/components/planner/quarters/courses/DraggableCourseCard.tsx

```diff
@@ -7,15 +7,17 @@ export interface DraggableCourseCardProps {
   course: StoredCourse;
   index: number;
   draggableId: string;
-  quarterId?: string;
   isCustom: boolean;
+  quarterId?: string;
+  customDeleteCourse?: () => void;
```

Comment:
Unnecessary Prop drilling. There should be a `handleRemoveCustom` function in the PlannerContext

Reply:
customDeleteCourse doesn't handle deleting a custom course, it handles unique deletion. This is because deleting draggable courses within RequirementsComponentEditing.tsx is different than deleting it from a planner. So I can't use handleRemoveCustom from PlannerContext